#
# Patch developed in-house. Solaris-specific; not suitable for upstream.
#
# Remove test 'test_ca' because it depends on directories
# not present in the build directory. The rest of tests are ok.
#
--- a/test/Makefile.orig	Thu Apr  2 12:11:12 2015
+++ b/test/Makefile	Thu Apr  2 12:11:21 2015
@@ -156,7 +156,7 @@
 	test_rand test_bn test_ec test_ecdsa test_ecdh \
 	test_enc test_x509 test_rsa test_crl test_sid \
 	test_gen test_req test_pkcs7 test_verify test_dh test_dsa \
-	test_ss test_ca test_engine test_evp test_evp_extra test_ssl test_tsa test_ige \
+	test_ss test_engine test_evp test_evp_extra test_ssl test_tsa test_ige \
 	test_jpake test_srp test_cms test_ocsp test_v3name test_heartbeat \
 	test_constant_time test_verify_extra test_clienthello test_sslv2conftest \
 	test_dtls test_bad_dtls
